openmrs/apps/home/extension.json

```diff
@@ -129,6 +129,15 @@
     "icon": "fa fa-calendar",
     "order": 14,
     "requiredPrivilege": "app:appointments"
+  },
+  "labLite": {
+    "id": "bahmni.lab",
+    "extensionPointId": "org.bahmni.home.dashboard",
+    "type": "link",
+    "translationKey": "MODULE_LABEL_LAB_ENTRY_KEY",
+    "url": "http://localhost:8200/lab/home",
```

Comment:
This URL is incorrect. Points to local. Needs to be "relative" to current server. This config gets deployed on QA/Demo environments (not just on local).

Reply:
resolved with the relative url : "/lab/home"